docs/en/observability/synthetics-run-test.asciidoc

```diff
@@ -1,147 +1,121 @@
+There are three ways to create a browser monitor: project monitors, {heartbeat}, and {agent}.
 
 [discrete]
-[[synthetics-test-suite]]
-== Use a test suite
-
-A test suite can contain any number of journeys in any number of files using the file extensions `.journey.ts` or `.journey.js`.
-You can use the `@elastic/synthetics` package as a library to run a test suite.
-
-To test a test suite locally, start by writing your tests:
-
-. Create a new https://docs.npmjs.com/cli/v7/commands/npm-init[NPM/Node.js project].
-. Create a `javascript` or `typescript` file that imports your tests (using the `.journey.ts` or `.journey.js` file extension).
-. Make sure any dependencies are defined with an `import` outside of the `journey` object.
-. Compile everything together.
-
-See an example of a test suite in the https://github.com/elastic/synthetics-demo[elastic/synthetics-demo] repository.
-The examples below show how to test that repository's `todo` directory locally.
-
-Clone the repository, change into the directory containing the synthetics test suite,
-and install the necessary dependencies including `@elastic/synthetics`:
-
-[source,sh]
-----
-# Check out the synthetics repo and included examples
-git clone git@github.com:elastic/synthetics-demo.git &&\
-cd synthetics-demo/todos/ &&\
-# Install all required dependencies for the todos example
-npm install
-----
-
-[NOTE]
-==============
-If you want to test your own suite locally, be sure to install the `@elastic/synthetics` package:
-
-[source,sh]
-----
-npm install -g @elastic/synthetics
-----
-==============
-
-From inside the directory containing the synthetics test suite, run the provided tests.
-By default, the `elastic-synthetics` runner will only run files matching the filename `*.journey.(ts|js)*`.
-
-[source,sh]
-----
-# Run tests on the current directory. The dot `.` indicates
-# that it should run all tests in the current directory.
-npx @elastic/synthetics .
-----
+[[synthetic-monitor-choose-project]]
+=== Project monitors
 
-[discrete]
-[[synthetics-test-suite-send]]
-=== Send to {stack}
+beta[]
 
-After testing locally, you can run the test suite using the Elastic Synthetics integration or {heartbeat}
-depending on <<uptime-set-up-choose,the approach you chose to set up>>.
+Use the `@elastic/synthetics` library's `push` command to create monitors.
+Pushing your project creates a new browser monitor in {kib} for each journey in the project. Using the `push` command allows you to manage all browser monitors using a GitOps workflow.
 
---
-include::tab-widgets/uptime-monitoring/run-test-suite-widget.asciidoc[]
---
+Alternatively, you could keep monitors in sync directly from your local development environment.
 
-[discrete]
-[[synthetics-inline-journey]]
-== Use an inline monitor
+// pros
+We recommend users to adopt a GitOps workflow when using project monitors because it allows teams to keep their monitors in sync with their project's `main` branch.
```

Comment:
```suggestion
We recommend users adopt a GitOps workflow when using project monitors because it allows teams to keep their monitors in sync with their project's `main` branch.
```